modules/administration-guide/examples/snip_che-technology-preview.adoc

```diff
@@ -0,0 +1,21 @@
+//To indicate that a feature is in Technology Preview, include the snippets/technology-preview.adoc file in the feature’s assembly or module to keep the supportability wording consistent across Technology Preview features. Provide a value for the :FeatureName: variable before you include this module.
+//e.g.:
+//:FeatureName: The XYZ plug-in
+//include::snippets/technology-preview.adoc[]
+//If you don't do this, the result will be an incorrect replacement.
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+{FeatureName} is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
```

Comment:
Consider removing mention to Red Hat production service level agreements (SLAs) in the upstream docs.